backend-rust/src/migrations/m0017-reward-metrics.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE metrics_rewards (
+    block_height        BIGINT NOT NULL,
+    block_slot_time     TIMESTAMPTZ NOT NULL,
+    account_index       BIGINT NOT NULL,
+    amount              BIGINT NOT NULL
+);
+
+INSERT INTO metrics_rewards
+WITH per_block AS (
+  SELECT
+    block_height,
+    account_index,
+    (SELECT slot_time FROM blocks WHERE height = block_height) AS block_slot_time,
+    SUM(amount) AS rewards
+  FROM account_statements
+  WHERE entry_type IN (
+    'FinalizationReward',
+    'FoundationReward',
+    'BakerReward',
+    'TransactionFeeReward'
+  )
+  GROUP BY block_height, account_index
+  HAVING COUNT(*) > 0
```

Comment:
Do we need this line? I don't see when this could be 0

Reply:
It is rather often 0. It is empty when there are no rewards.